src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -9,15 +9,19 @@
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import org.hibernate.annotations.Filter;
+import org.hibernate.annotations.FilterDef;
+import org.hibernate.annotations.ParamDef;
 import org.hibernate.annotations.SQLDelete;
-import org.hibernate.annotations.Where;
 
 @Entity
 @Getter
 @Setter
+@FilterDef(name = "isDeletedFilter",
+        parameters = @ParamDef(name = "isDeleted", type = Boolean.class))
 @Table(name = "books")
 @SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
-@Where(clause = "is_deleted=false")
+@Filter(name = "isDeletedFilter", condition = "is_deleted = false")
```

Comment:
```suggestion
@SQLRestriction(name = "isDeletedFilter", condition = "is_deleted = false")
```

Reply:
It doesn't work with @SQLRestriction. It's recommended to use @Filter. [https://www.baeldung.com/spring-jpa-soft-delete](url).

Reply:
got it